src/test/components/UrlManager.test.js

```diff
@@ -230,6 +233,45 @@ describe('UrlManager', function() {
     expect(getCurrentSearchString(getState())).toBe('B');
   });
 
+  it('allows navigating back and forward when moving between content pages', async () => {
+    // The test will start at the home.
+    const urlPath = '/ ';
+    const {
+      getState,
+      createUrlManager,
+      waitUntilUrlSetupPhase,
+      dispatch,
+    } = setup(urlPath);
+    createUrlManager();
+
+    await waitUntilUrlSetupPhase('done');
+    expect(getDataSource(getState())).toBe('none');
+    expect(window.history.length).toBe(1);
+
+    // Now click on the "all recordings" link.
```

Comment:
nit: Describing these as "click" events is a bit incongruous as these are not a real click event.

Reply:
I improved all comments, this should be clearer now.